src/hyp3_isce2/metadata/templates/insar_burst/readme.md.txt.j2

```diff
@@ -0,0 +1,306 @@
+* HyP3 ISCE2 plugin, DOI: [10.5281/zenodo.8007397](https://doi.org/10.5281/zenodo.8007397)
+
+Refer to https://github.com/isce-framework/isce2#readme for more information about ISCE2 software.
+
+----------------
+*Consider opening this document in a Markdown editor/viewer for easier reading.*
+----------------
+Interferometric SAR (InSAR) uses the phase differences from repeat passes over the same area to identify regions where
+the distance between the sensor and the earth's surface has changed. This allows for the detection and quantification
+of deformation or movement.
+
+These burst-based products are generated from pairs of individual bursts acquired by Sentinel-1. ESA packages
+collections of bursts into SLC products for distribution. These SLCs cover large areas, resulting in large file sizes.
+In cases where only a small area of the SLC is of interest, extracting individual bursts from the SLC allows InSAR
+processing to be performed much more quickly and efficiently than processing an entire SLC pair.
+
+In addition to the efficiencies provided by the smaller file size, Sentinel-1 bursts are consistent in their footprint.
+The framing of SLCs is not always consistent along a path, so bursts may be included at the top of one SLC for one
+pass and the bottom of the next SLC for a different pass. This makes it more difficult to find and process all the
+acquisitions available for a specific burst over time. When searching through time at the burst level, you can be
+assured that your stack includes all the acquisitions for that burst footprint.
+
+Note that the imaging surface will vary depending on the sensor wavelength. Because Sentinel-1 is a C-band sensor,
+the waves will not penetrate very deeply into vegetation. The imagery likely represents the top of the canopy in
+densely vegetated areas rather than the actual terrain. In addition, vegetated areas tend to have low coherence,
+because plants can grow or move from one acquisition to the next. Use caution when generating interferograms for
+areas with extensive/dense vegetation cover.
+
+A digital elevation model is required for processing InSAR. The DEM used for this particular product is the
+Copernicus GLO-30 Public DEM, which has a native resolution of 1 arc second (about 30 meters). This DEM is a
+global Digital Surface Model (DSM) derived from the WorldDEM. The WorldDEM is based on radar satellite data
+acquired by the TanDEM-X mission, which was funded by the German Aerospace Center (DLR) and Airbus Defence and Space,
+and edited to flatten water bodies, provide consistent flow of rivers, and apply corrections to shore/coastlines and
+special features. For an overview of the dataset, visit
+https://spacedata.copernicus.eu/collections/copernicus-digital-elevation-model.
+
+***************
+# Product Contents #
+
+The files generated in this process include:
+
+1. Wrapped Interferogram (GeoTIFF)
+2. Unwrapped Interferogram (GeoTIFF, PNG image)
```

Comment:
Are we including browse images for the interferograms? I was surprised to see that there's a PNG for the unwrapped but not the wrapped; I would have expected it the other way around.

Reply:
@forrestfwilliams I think you're best suited to answer this.

Reply:
We're currently just including grayscale PNGs for unwrapped interferograms.